platform/MCU/nRF52840/inc/platform_config.h

```diff
@@ -101,9 +101,9 @@
 //RGB LEDs
 #define LED_MIRROR_SUPPORTED                        1
 #define LEDn                                        (4 * (LED_MIRROR_SUPPORTED + 1))
-#define LED_RED                                     LED2  // RED Led
-#define LED_GREEN                                   LED3  // GREEN Led
-#define LED_BLUE                                    LED4  // BLUE Led
+#define LED_RED                                     PARTICLE_LED2  // RED Led
```

Comment:
We could also potentially prefix all of these as they are somewhat too generic, same for the other `platform_config.h`